reconcile/terraform_cloudflare_users.py

```diff
@@ -0,0 +1,298 @@
+import sys
+from dataclasses import dataclass
+from typing import (
+    Any,
+    Iterable,
+    Mapping,
+    MutableMapping,
+    Optional,
+    Set,
+    Tuple,
+)
+
+from reconcile import queries
+from reconcile.gql_definitions.terraform_cloudflare_users import (
+    terraform_cloudflare_roles,
+)
+from reconcile.gql_definitions.terraform_cloudflare_users.terraform_cloudflare_roles import (
+    CloudflareAccountRoleQueryData,
+    CloudflareAccountRoleV1,
+)
+from reconcile.status import ExitCodes
+from reconcile.utils import gql
+from reconcile.utils.external_resource_spec import ExternalResourceSpec
+from reconcile.utils.runtime.integration import QontractReconcileIntegration
+from reconcile.utils.secret_reader import SecretReader
```

Comment:
Lets use `SecretReaderBase`. `SecretReader` is marked for deprecation as it is untyped. For backwards-compat, `SecretReader` implements `SecretReaderBase`.